frontend/public/components/utils/k8s-watch-hook.ts

```diff
@@ -61,6 +61,9 @@ const useModelsLoaded = (): boolean => {
   return ref.current;
 };
 
+export const useModelLoaded = (model: string): boolean =>
```

Comment:
+1 hook might be useful

Reply:
You can use https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/hooks/useK8sModel.ts instead

Reply:
yeah that sound much better